sort.py

```diff
@@ -0,0 +1,22 @@
+
+def sort(array):
+    if not array or len(array) <= 1:
+        return array # nothing to sort
+
+    index = 0
+    swapped = True
+    while index < len(array) and swapped:
+        swapped = False
```

Comment:
If we change swapped to False here and it doesn't go into the if block, do we want to stop checking the other elements?